src/options.rs

```diff
@@ -73,6 +73,8 @@ pub struct Options {
     help = "Do not index inscriptions."
   )]
   pub(crate) no_index_inscriptions: bool,
+  #[arg(long, help = "Use search index at <SEARCH_INDEX>.")]
+  pub(crate) search_index: Option<PathBuf>,
```

Comment:
I think this doesn't even need to be configurable initially. We could just put it into the ord data dir.